2-mandatory/4-people-i-know.js

```diff
@@ -378,14 +378,22 @@ First, I want you to find all of my friends who are 35 or older.
 */
 
 let thirtyFiveOrOlder = [];
-
+friends.forEach((friend) => {
```

Comment:
This works.. but you could also use friends.filter as this will also return an Array